presto-docs/src/main/sphinx/release/release-0.258.rst

```diff
@@ -0,0 +1,44 @@
+=============
+Release 0.258
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix issues in SQL functions where query might fail with compiler error when function implementation includes lambda variable that has the same name as column names or function input names.
+* Fixes a bug in Parquet dereference pushdown. The issue causes column name not picked up properly, resulting in inconsistent query result in certain cases.
+* Add Cauchy distribution CDF :func:`cauchy_cdf` and inverse CDF :func:`inverse_cauchy_cdf` functions to MathFunctions.
+* Add SQL functions :func:`array_dupes` and :func:`array_as_dupes`.
+* Add additional details to memory exceeded error messages to simplify debugging. Disabled by default. Can be enabled by setting the ``verbose_exceeded_memory_limit_errors_enabled`` session property to ``true``.
+* Allow dynamic filtering for large number of collected values by converting the predicate into a range.
+* Support dynamic filtering with comparison operators.
+
+Iceberg Connector Changes
+_________________________
+* Add ORC support for iceberg connector.
+
+Cassandra Changes
+________________
+* Improve cassandra query performance by adding meta cache to avoid too many remote meta operations to the cassandra server.
+
+Elasticsearch Changes
+_____________________
+* Fix incorrect pushdown if ``IS NULL`` predicate in Elasticsearch.
+
+Hive Changes
+____________
+* Add support for allowing to match columns between table and partition schemas by names for HUDI tables. This is enabled when configuration property ``hive.parquet.use-column-names`` or the hive catalog session property ``parquet_use_column_names`` is set to ``true``. By default they are mapped by index.
```

Comment:
I'm confused about this note. Is this change specific to HUDI?  Like was the parquet feature was there before but it didn't work for HUDI tables or did they add this feature for parquet tables for the first time, and it happens to be that HUDI tables use parquet so also benefit?

Reply:
Looks like partition schema support for parquet was already added earlier as part of this pr: https://github.com/prestodb/presto/pull/16011. 
And this one is adding the support for HUDI tables. Hope that clarifies it.